src/auto-evo/RunResults.cs

```diff
@@ -508,23 +508,24 @@ public List<Species> GetNewSpecies()
         /// <summary>
         ///   Prints to log a summary of the results
         /// </summary>
-        public void PrintSummary(PatchMap previousPopulations = null)
+        public void PrintSummary(GameWorld world, PatchMap previousPopulations = null)
         {
             GD.Print("Start of auto-evo results summary (entries: ", results.Count, ")");
 
-            GD.Print(MakeSummary(previousPopulations));
+            GD.Print(MakeSummary(world, previousPopulations));
 
             GD.Print("End of results summary");
         }
 
         /// <summary>
         ///   Makes summary text
         /// </summary>
+        /// <param name="world">The current global game data</param>
```

Comment:
These need to be updated, because this is not purely a summary making code anymore, this is actually now impacting the state of the GameWorld object.